pom.xml

```diff
@@ -279,6 +279,12 @@
         <artifactId>gson</artifactId>
         <version>2.8.0</version>
     </dependency>
+
+    <dependency>
```

Comment:
Looks like we might need to add a line in the instructions for the linux/mac users. We used Tess4j in the project (http://tess4j.sourceforge.net/usage.html). which points to the original tesseract-ocr github page (https://github.com/tesseract-ocr/tesseract/wiki) that suggests running "sudo port install tesseract".

Although to be honest I'm not able to test since I don't have access to a linux device. Let me know if there is anything I can do and what you guys find out.